cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_train_dense_default_impl.i

```diff
@@ -187,20 +188,23 @@ private:
         if (!isPositive<algorithmFPType, cpu>(imp.var)) imp.var = 0; //roundoff error
     }
 
-    static void calcPrevImpurity(ImpurityData & imp, ClassIndexType x, size_t n, size_t l)
+    // TODO: check for weight
```

Comment:
TODO

Reply:
It'll be removed after checking appropriate formula. It'll be done later.